### DISCUSSION TITLE
"Error 503. Service Unavailable" on Cpanel of Symfony 6

### DISCUSSION THREAD

**Discussion by MalayRoy** [symfony/symfony#47544] [Q&A]
### Symfony version(s) affected

6.1.3

### Description

I have developed a project with Symfony 6 and PHP 8.1.5. It's working properly on my dev pc. When PHP version 8.1 upgrade on Cpanel and deploy my project then show an error :

"Error 503. Service Unavailable.
The server is temporarily unavailable. Please check back shortly."

URL: https://biswasexpress.com/public/index.php

Image :
![Cpanel Error](https://user-images.githubusercontent.com/42863283/189526430-a0601913-fbfe-42bf-a215-3c3910ff8c9d.JPG)

How to solve this error. 
Thanks...

### How to reproduce

"Error 503. Service Unavailable" on Cpanel of Symfony 6

**Comment by lyrixx** [comment]
You may search in application logs for more information
